antenna/throttler.py

```diff
@@ -277,86 +277,79 @@ def match_unsupported_product(throttler, data):
         condition=match_unsupported_product,
         percentage=None
     ),
 
-    # 100% of crashes that have a comment
+    # Accept 100% of crash reports submitted through about:crashes
+    Rule(
+        rule_name='throttleable_0',
+        key='Throttleable',
+        condition=lambda throttler, x: x == '0',
+        percentage=100
+    ),
+
+    # Accept 100% of crash reports that have a comment
     Rule(
         rule_name='has_comments',
         key='Comments',
         condition=always_match,
         percentage=100
     ),
 
-    # 100% of crashes that have an email address with at least an @
+    # Accept 100% of crash reports that have an email address with at least an @
     Rule(
         rule_name='has_email',
         key='Email',
         condition=lambda throttler, x: x and '@' in x,
         percentage=100
     ),
 
-    # 100% of all ReleaseChannel=aurora, beta, esr channels
+    # Accept 100% of crash reports in ReleaseChannel=aurora, beta, esr channels
     Rule(
         rule_name='is_alpha_beta_esr',
         key='ReleaseChannel',
         condition=lambda throttler, x: x in ('aurora', 'beta', 'esr'),
         percentage=100
     ),
 
-    # 100% of all ReleaseChannel=nightly
+    # Accept 100% of crash reports in ReleaseChannel=nightly
     Rule(
         rule_name='is_nightly',
         key='ReleaseChannel',
         condition=lambda throttler, x: x.startswith('nightly'),
         percentage=100
     ),
 
-    # 20% of Firefox 56 and earlier
+    # Accept 20%, reject 80% of Firefox 56 and earlier (all channels)
     Rule(
         rule_name='firefox_pre_57',
         key='*',
         condition=match_firefox_pre_57,
         percentage=20
     ),
 
-    # 10% of ProductName=Firefox
+    # Accept 10%, reject 90% of Firefox desktop release channel
     Rule(
         rule_name='is_firefox_desktop',
-        key='ProductName',
-        condition=lambda throttler, x: x == 'Firefox',
+        key='*',
+        condition=lambda throttler, data: (
+            data.get('ProductName', '') == 'Firefox' and
+            data.get('ReleaseChannel', '') == 'release'
+        ),
         percentage=10
     ),
 
-    # 100% of ProductName=Fennec
-    Rule(
-        rule_name='is_fennec',
-        key='ProductName',
-        condition=lambda throttler, x: x == 'Fennec',
-        percentage=100
-    ),
-
-    # 100% of all Version=alpha, beta or special
+    # Accept everything else
     Rule(
-        rule_name='is_version_alpha_beta_special',
-        key='Version',
-        condition=lambda throttler, x: '.' in x and x[-1].isalpha(),
-        percentage=100
-    ),
-
-    # 100% of ProductName=Thunderbird & SeaMonkey
-    Rule(
-        rule_name='is_thunderbird_seamonkey',
-        key='ProductName',
-        condition=lambda throttler, x: x and x[0] in 'TSC',
+        rule_name='accept_everything',
+        key='*',
+        condition=always_match,
         percentage=100
-    ),
-
-    # Reject everything else
+    )
```

Comment:
I added this "accept everything" catch all since it means that all "how does the Mozilla Socorro collector throttle?" questions can be answered solely by looking at this rule set.